scaling_performance/optimizing_storage.adoc

```diff
@@ -30,46 +30,39 @@ a|* Presented to the operating system (OS) as a block device
 bypassing the file system
 * Also referred to as a Storage Area Network (SAN)
 * Non-shareable, which means that only one client at a time can mount an endpoint of this type
-| CNS/CRS GlusterFS <1> iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, AWS EBS <1>, Dell/EMC Scale.IO,
-VMware vSphere Volume, GCE Persistent Disk <1>, Azure Disk
+| CNS/CRS GlusterFS footnoteref:[dynamic PV,CNS/CRS GlusterFS, Ceph RBD, OpenStack Cinder, AWS EBS, Azure Disk, GCE persistent disk, and VMware vSphere support dynamic persistent volume (PV) provisioning natively in {product-title}.]  iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, AWS EBS footnoteref:[dynamic PV], Dell/EMC Scale.IO, VMware vSphere Volume, GCE Persistent Disk footnoteref:[dynamic PV], Azure Disk
```

Comment:
Hey @tmorriso-rh and @openshift/team-documentation - it seems that the portal doesn't support space in the footnoteref. So this should be dynamicPV and not dynamic PV. I have made the fix for this and other footnotes via https://github.com/openshift/openshift-docs/pull/6597.

@tmorriso-rh - can you please add this in our docs guidelines?

Reply:
@vikram-redhat Hi Vikram, I added this information to the quick markup reference table. Please revise as necessary. Thanks.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference